python/dgl/graphbolt/graph_storage/csc_sampling_graph.py

```diff
@@ -222,7 +222,53 @@ def sample_neighbors(
         # Ensure nodes is 1-D tensor.
         assert nodes.dim() == 1, "Nodes should be 1-D tensor."
         assert fanout >= 0 or fanout == -1, "Fanout shoud have value >= 0 or -1"
-        return self._c_csc_graph.sample_neighbors(nodes, fanout, replace)
+        return self._c_csc_graph.sample_neighbors(nodes, [fanout], replace)
+
+    def sample_etype_neighbors(
```

Comment:
Why don't we merge this API with sample_neighbors?

Reply:
1. This is compatible with current interface.
2. `Fanout `is int and tensor in 2 cases.